client/i18n/locales/chinese/translations.json

```diff
@@ -1,444 +1,444 @@
-    "msg-25": "We need your name so we can put it on your certification. Add your name to your account settings and click the save button. Then we can issue your certification.",
-    "msg-26": "It looks like you have not completed the necessary steps. Please complete the required projects to claim the {{name}} Certification.",
-    "msg-27": "It looks like you already have claimed the {{name}} Certification",
-    "msg-28": "@{{username}}, you have successfully claimed the {{name}} Certification! Congratulations on behalf of the freeCodeCamp.org team!",
-    "msg-29": "Something went wrong with the verification of {{name}}, please try again. If you continue to receive this error, you can send a message to support@freeCodeCamp.org to get help.",
-    "msg-30": "Error claiming {{certName}}",
-    "msg-31": "We could not find a user with the username \"{{username}}\"",
-    "msg-32": "This user needs to add their name to their account in order for others to be able to view their certification.",
-    "msg-33": "This user is not eligible for freeCodeCamp.org certifications at this time.",
-    "msg-34": "{{username}} has chosen to make their portfolio private. They will need to make their portfolio public in order for others to be able to view their certification.",
-    "msg-35": "{{username}} has chosen to make their certifications private. They will need to make their certifications public in order for others to be able to view them.",
-    "msg-36": "{{username}} has not yet agrees to our Academic Honesty Pledge.",
-    "msg-37": "It looks like user {{username}} is not {{cert}} certified",
-    "msg-38": "That does not appear to be a valid challenge submission",
-    "msg-39": "You have not provided the valid links for us to inspect your work.",
-    "msg-40": "No social account found",
-    "msg-41": "Invalid social account",
-    "msg-42": "No {{website}} account associated",
-    "msg-43": "You've successfully unlinked your {{website}}",
-    "msg-44": "Check if you have provided a username and a report",
-    "msg-45": "A report was sent to the team with {{email}} in copy"
+    "msg-1": "申请认证之前，你必须先接受我们的《学术诚信条例》",
+    "msg-2": "出现了一些奇怪的情况。如果再出现这种情况，请考虑在 https://github.com/freeCodeCamp/freeCodeCamp/issues/new 提 issue",
+    "msg-3": "有些不对劲。已生成报告，通知 freeCodeCamp.org 团队。",
+    "msg-4": "出了点问题，请检查并重试。",
+    "msg-5": "你的账号已被成功删除",
+    "msg-6": "你的进度已被重置",
+    "msg-7": "你无权继续访问",
+    "msg-8": "我们未找到你搜索的信息。请检查并重试。",
+    "msg-9": "更新你的账号出了点问题。请检查并重试。",
+    "msg-10": "我们已更新你的偏好设置。",
+    "msg-11": "邮箱格式无效",
+    "msg-12": "currentChallengeId 是无效的挑战 ID",
+    "msg-13": "主题无效",
+    "msg-14": "主题已设置",
+    "msg-15": "你的主题已更新！",
+    "msg-16": "用户名已和此账号关联",
+    "msg-17": "用户名已和别的账号关联",
+    "msg-18": "我们已更新你的用户名为 {{username}}",
+    "msg-19": "你的账号未退出，请稍后重试。",
+    "msg-20": "链接中的邮箱格式不正确",
+    "msg-21": "有点不对劲。请刷新链接再登录/注册",
+    "msg-22": "你点击的链接似乎已过期，请刷新链接再登录",
+    "msg-23": "成功了！你已经登录账号。Happy Coding!",
```

Comment:
```suggestion
    "msg-23": "成功了！你已经登录账号。",
```

Not sure.

Reply:
emmm I always use "Happy Coding" in articles. But ok, we can delete it here.